.github/workflows/release.yml

```diff
@@ -387,7 +361,7 @@ jobs:
           git checkout $BRANCH_NAME
           
           # Update deploy manifest 
-          if [[ $env.IS_RC ]]; then 
+          if [[ $IS_RC == 1 ]]; then 
             sed -i "s/perconalab\/everest.*/perconalab\/everest:$VERSION/g" deploy/quickstart-k8s.yaml
             sed -i "s/percona\/percona-everest.*/perconalab\/everest:$VERSION/g" deploy/quickstart-k8s.yaml
           else
```

Comment:
btw this might have been one of the reason of the mistakenly used `perconalab` vs `percona`

Reply:
what the `sed` command does still needs to be re-checked, but the fact is that the condition was wrong